cs/src/comm/epoxy-transport/EpoxyConnection.cs

```diff
@@ -194,21 +194,17 @@ internal static Frame MessageToFrame(ulong conversationId, string methodName, Pa
                 const int initialLayerDataBufferSize = 150;
                 var outputBuffer = new OutputBuffer(initialLayerDataBufferSize);
                 var compactWriter = new CompactBinaryWriter<OutputBuffer>(outputBuffer);
-                // TODO: See TODO below about issues with IBonded Marshal.TO(...)
                 compactWriter.WriteVersion();
                 layerData.Serialize(compactWriter);
                 frame.Add(new Framelet(FrameletType.LayerData, outputBuffer.Data));
             }
 
             {
-                var userData = payload.IsError ? (IBonded)payload.Error : (IBonded)payload.RawPayload;
-
+                var userData = message.IsError ? (IBonded)message.Error : (IBonded)message.RawPayload;
 
                 const int initialPayloadBufferSize = 1024;
                 var outputBuffer = new OutputBuffer(initialPayloadBufferSize);
                 var compactWriter = new CompactBinaryWriter<OutputBuffer>(outputBuffer);
-                // TODO: marshal dies on IBonded Marshal.To(compactWriter, request)
```

Comment:
The code below:

```
compactWriter.WriteVersion();
userData.Serialize(compactWriter);
```

is manually marshalling because `Marshal.To` throws. That's what the original comment was supposed to capture.